pkg/peertls/tlsopts/tls.go

```diff
@@ -15,8 +18,14 @@ import (
 	"storj.io/storj/pkg/storj"
 )
 
+// TransportCredentials wraps an embedded credentials.TransportCredentials.
+// It decorates the `ClientHandshake` method.
+type TransportCredentials struct {
```

Comment:
Should this be in `pkg/transport` instead?

Reply:
:+1: